backend/clubs/views.py

```diff
@@ -2895,6 +2895,14 @@ def get(self, request):
         return Response({"count": ans})
 
     def post(self, request):
+        if "HTTP_AUTHORIZATION" in request.META:
```

Comment:
This is a security issue. You can bypass the security check by just not including the authorization header.